src/parsing/abstract-parser.h

```diff
@@ -2028,15 +2028,15 @@ void AbstractParser<Impl>::ParseFunction(
     auto elapsed = std::chrono::high_resolution_clock::now() - start;
     long long microseconds =
         std::chrono::duration_cast<std::chrono::microseconds>(elapsed).count();
-    printf("Deserialized function literal for '");
+    printf("\nDeserialized function literal for '");
     if (literal->has_shared_name()) {
       for (const AstRawString* s : literal->raw_name()->ToRawStrings()) {
         printf("%.*s", s->byte_length(), s->raw_data());
       }
     }
-    printf("' in %lld us: %p\n", microseconds, literal);
+    printf("' in %lld us\n", microseconds);
     // TODO(binast): Store the literal on the ParseInfo
-    result = literal;
+    // result = literal;
```

Comment:
👀  was this intentional?

Reply:
It was...however I could be convinced that it's a bad idea haha. It was done to be able to measure both deserialization and re-parsing time back to back. Maybe something a little more principled in a separate diff?

Reply:
goootcha. yeah, this seems fine for now then